src/NearOrg/Notifications/Notification.jsx

```diff
@@ -322,5 +366,24 @@ return (
         {actionable && <Button href={postUrl}>View</Button>}
       </Right>
     </Content>
+    <Widget
+      src="${REPL_ACCOUNT}/widget/DIG.DropdownMenu"
+      props={{
+        trigger: <i className="ph-bold ph-dots-three" />,
+        items: buildMenu(),
+      }}
+    />
+    <Widget
+      src="${REPL_ACCOUNT}/widget/DIG.Dialog"
+      props={{
+        type: "alert",
+        title: <modalContent />,
```

Comment:
I think we can avoid this function and use this instead:
```suggestion
        title: `Do you want to stop receiving push notification for ${type}`,
```

Reply:
This also has been addressed in the latest commit. Thank you